stable/dynamic-gateway-service/values.yaml

```diff
@@ -119,6 +119,9 @@ datapower:
   snmpLocalAddress: 0.0.0.0
   snmpPort: 1161
   flexpointBundle:
+  loggingService:
```

Comment:
It just occurred to me that a customer may want to set more than one entry here. Could/Should this be an array instead? This would be more future-proof